migration/fluentd-wikigridd-to-bitnami.md

```diff
@@ -0,0 +1,14 @@
+# Upgrade to fluentd chart from wikigrid to bitnami
+
+The bitnami chart is preferred over wikigrid as it  provides a possibility to create fluentd service using a single chart for the aggregator and forwarder
+The following snippet shows the commands nessecary to remove the old `fluentd` releasee, and how to install the new `fluentd` service.
+
+```
+# Delete old release
+./bin/ck8s ops helmfile sc -l app=fluentd destroy
+
+
+# Install new release
+./bin/ck8s ops helmfile sc -l app=fluentd  apply
```

Comment:
I think you have missed some things here. You also need to destroy the `app=fluentd-aggregator` release, which must be done with helm instead of helmfile if you are doing this after checking out this release (since the chart is removed from helmfile it cannot manage it, but helm can. Let me know if you have any questions regarding this).
You also must upgrade fluentd in the workload cluster. Is it enough to run `ck8s apply wc` or does one need some specific instructions for that. If one needs specific instructions, then add them here.

Reply:
> I think you have missed some things here. You also need to destroy the `app=fluentd-aggregator` release, which must be done with helm instead of helmfile if you are doing this after checking out this release (since the chart is removed from helmfile it cannot manage it, but helm can. Let me know if you have any questions regarding this).
> You also must upgrade fluentd in the workload cluster. Is it enough to run `ck8s apply wc` or does one need some specific instructions for that. If one needs specific instructions, then add them here.

`ck8s apply wc` is enough for wc.

Reply:
That's good :slightly_smiling_face: 

Though one still needs to destroy the `app=fluentd-aggregator` release. Please add an instruction for that